src/components/SequenceInput/SequenceInput.js

```diff
@@ -33,6 +37,11 @@ export default function SequenceInput({
   const [playerName, setPlayerName] = useState('')
   const [timeCodeLowerThirdIn, setTimeCodeLowerThirdIn] = useState('')
   const [timeCodeLowerThirdLength, setTimeCodeLowerThirdLength] = useState('')
+  const inputOnUpdate = useRef(null)
+
+  useEffect(() => {
+    isEmpty && resetState()
+  }, [isEmpty, isAlert])
```

Comment:
Do you need "isAlert" in the dependency array? It doesn't seem to influence the function body.

Reply:
good catch.